console/src/ui/events.rs

```diff
@@ -114,14 +90,36 @@ impl EventList {
     }
 
     fn style_event(&self, i: usize, entry: &EventEntry) -> Vec<Text<'_>> {
-        let level = match *entry.level() {
-            Level::INFO => Text::styled(" INFO ", Style::default().fg(Color::White)),
-            Level::DEBUG => Text::styled("DEBUG ", Style::default().fg(Color::LightCyan)),
-            Level::ERROR => Text::styled("ERROR ", Style::default().fg(Color::Red)),
-            Level::TRACE => Text::styled("TRACE ", Style::default().fg(Color::Green)),
-            Level::WARN => Text::styled(" WARN ", Style::default().fg(Color::Yellow)),
+        let level = match entry.level() {
+            None => Text::styled(" NONE ", Style::default().fg(Color::White)),
+            Some(Level::Info) => Text::styled(" INFO ", Style::default().fg(Color::White)),
+            Some(Level::Debug) => Text::styled("DEBUG ", Style::default().fg(Color::LightCyan)),
+            Some(Level::Error) => Text::styled("ERROR ", Style::default().fg(Color::Red)),
+            Some(Level::Trace) => Text::styled("TRACE ", Style::default().fg(Color::Green)),
+            Some(Level::Warn) => Text::styled(" WARN ", Style::default().fg(Color::Yellow)),
         };
-        let text = format!("{}\n", entry.display());
+        let mut text = String::new();
+        let mut first = true;
+        for value in &entry.event.values {
+            if first {
+                first = false;
+            } else {
+                text.push_str(", ");
+            }
+            if let Some(field) = &value.field {
+                write!(text, r#"{}("#, field.name).unwrap();
+                match &value.value {
+                    Some(value::Value::Signed(i)) => write!(text, "{}", i).unwrap(),
+                    Some(value::Value::Unsigned(u)) => write!(text, "{}", u).unwrap(),
+                    Some(value::Value::Boolean(b)) => write!(text, "{}", b).unwrap(),
+                    Some(value::Value::Str(s)) => write!(text, "{}", s).unwrap(),
+                    Some(value::Value::Debug(d)) => write!(text, "{}", d.debug).unwrap(),
+                    None => {}
+                }
+                text.push_str(r#"")"#);
```

Comment:
Looks like an extra quotation mark slipped in here --- when I run the console, all event fields render like this:
```
│DEBUG message(hello! I'm gonna shave a yak."), excitement(yay!")
│TRACE message(yak shaved successfully")
│TRACE yak(1"), shaved(true")
│TRACE yaks_shaved(1")
```

I think the solution is just to remove it
```suggestion
                text.push_str(r#")"#);
```